lib/block/transaction_pool.go

```diff
@@ -0,0 +1,109 @@
+package block
+
+import (
+	"fmt"
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/errors"
+	"boscoin.io/sebak/lib/storage"
+	"boscoin.io/sebak/lib/transaction"
+)
+
+// TransactionPool is not `transaction.Pool`; this contains the all the valid
```

Comment:
I think that the name is confusing. ㅠㅠ

Reply:
@anarcher Hehe, I think so. Please make new name :)

Reply:
TxMessagePool? hehe